roles/traefik/tasks/main.yml

```diff
@@ -8,6 +8,29 @@
     - traefik
     - bootstrap
 
+- name: create directory for local traefik service override
+  sudo: yes
+  file:
+    dest:  /etc/systemd/system/traefik.service.d
+    state: directory
+    mode: 0755
+  tags:
+    - traefik
+    - bootstrap
+
+- name: create local traefik service override
+  sudo: yes
+  copy:
+    dest: /etc/systemd/system/traefik.service.d/marathon.conf
+    content: |
+      [Service]
+      ExecStartPre=-/usr/bin/smlr {{ traefik_marathon_endpoint }}/v2/leader
```

Comment:
Well, I didn't even know that that's what that syntax meant :) I think you're right, in that it won't end up mattering, and the scenario where it starts is better than not starting.